src/palette/elements/Tabs/ContentTabs.tsx

```diff
@@ -0,0 +1,39 @@
+import { Tab, TabsProps } from "palette/elements/Tabs"
+import React, { useState } from "react"
+import { LayoutRectangle } from "react-native"
+import { TabBarContainer } from "./TabBarContainer"
+
+/**
+ * Renders a  scrollable list of tabs. Tabs are not evenly spaced across screen
+ */
+export const ContentTabs: React.FC<TabsProps> = ({ onTabPress, activeTab, tabs }) => {
+  const [tabLayouts, setTabLayouts] = useState<Array<LayoutRectangle | null>>(tabs.map(() => null))
```

Comment:
Not sure if we have some guidelines on this, but what do you think of adding a blank line here as a separator? Maybe it's just me (or my personal preferences), but I have difficulties reading code when different context aren't separated

```suggestion
  const [tabLayouts, setTabLayouts] = useState<Array<LayoutRectangle | null>>(tabs.map(() => null))

```

Reply:
@olerichter00 Haha 😂. Ok I will add a new line.

Reply:
Thanks :)